components/vtpm2-server/vtpm2-server.yaml

```diff
@@ -28,12 +28,12 @@ spec:
             protocol: TCP
             hostPort: 5000
           volumeMounts:
-          - mountPath: /host/sockets
+          - mountPath: /host/tsi-secure/sockets
             name: tsi-sockets
       volumes:
       - name: tsi-sockets
         hostPath:
           # directory location on host
-           path: /var/tsi-secure/sockets
+           path: /var/sockets
```

Comment:
hmm, should this be consistent with the above? why is it different here? There are also inconsistencies with `vtpm1.yaml`.

Reply:
vtpm1.yaml has to be retired.